portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase

```diff
@@ -214,6 +214,71 @@ definition {
 			userEmailAddress = "test@liferay.com");
 	}
 
+	@description = "This is a use case for LPS-105151."
+	@priority = "4"
+	test OpenIDConnectOnInstanceLevelSmoke {
+		property dummy.socket.proxy.disabled = "true";
+		property test.name.skip.portal.instance = "Login#OpenIDConnectOnInstanceLevelSmoke";
+
+		var openIDConnectClientID = "48980943835-sk374ituib2k9ajdeobo5i7v54tt8nfe.apps.googleusercontent.com";
+		var openIDConnectClientSecret = "XWM-9_qRoQxBvFf3PUw_DTWC";
```

Comment:
We should never expose credentials in our Github repo. See how we deal we grab private credentials in our [Google SSO test](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase#L158-L159).

[This](https://issues.liferay.com/browse/LRCI-1457) is an example of how we request credentials to be added on Jenkins. In this case we can update one of the existing `google.client.id`/`google.client.secret` credentials on Jenkins.